html/views/common.js

```diff
@@ -20,6 +20,7 @@ function jammer(k, v) {
 	};
 	$(".Team" + id + " .Lead").toggleClass("HasLead", (leadJammer && !starPass));
 	$(".Team" + id).toggleClass("HasJammerName", (jn != ""));
+	$(".Team" + id).toggleClass("starPass", (starPass != false));
```

Comment:
Did you forget to push?

Reply:
No, I changed
`$(".Team" + id).toggleClass("IsStarPass ", (starPass != false));` to
`$(".Team" + id).toggleClass("starPass", (starPass != false));`
and removed the redundant class IstStarPass in the css of the html.

Reply:
You can still simplify to `starPass`

Reply:
Ok, got your point.